backend/serialization/vanilla-LibBackend-Op-AddOpEventV1-simple.json

```diff
@@ -622,7 +622,7 @@
           ]
         ],
         "spec": [
-          "HTTP",
+          "HTTPLegacy",
```

Comment:
Depending on whether this is unserialized, it may need to read the old types as well as the new types. I think it almost certainly does.

Reply:
Can you please clarify here?
My best guess/understanding is that `client` will have to adjust to handle both "HTTPLegacy" and "HTTP" when decoding, but unsure if that's what you mean, or something else.